src/cookieSync.js

```diff
@@ -267,6 +267,7 @@ function getStringifiedData(endPointArgs) {
     }
   };
   if(BIDDER_ARGS) data.bidders = BIDDER_ARGS;
+  data['account'] = ACCOUNT;
```

Comment:
I recommend keeping the pattern of `if(ACCOUNT) data.account = ACCOUNT`.

- The `if` statement will avoid setting the parameter to null or empty. 
- The syntax `data['account']` is equivalent to `data.account` and it looks like the latter is preferred here.

^ same for cookieSyncWithConsent.js